src/spatialmath/vector3.rs

```diff
@@ -111,7 +111,6 @@ mod tests {
         assert_eq!(vector.x, 1.0);
         assert_eq!(vector.y, 1.0);
         assert_eq!(vector.z, 1.0);
-        assert!(!vector.is_normalized())
```

Comment:
If this line is removed, the test it's in doesn't test anything. Was this on purpose?

Reply:
no, it wasn't, this was a mistake

Reply:
oh it wasn't a mistake, this doesn't belong here. It's tested later on